### PR TITLE
Feature Social Login

### PR DESCRIPTION
**Pull request by Zann5547**
We can leave additional comment about the code here.

### CODE REVIEW

File: file1.txt [hunk 1]
```diff
@@ -1 +1 @@
-hello
\ No newline at end of file
+hello world
```
Comment (Estyaque) [inline file1.txt:1]: Everything's fine but capitalized them and do another commit

Comment (Estyaque) [inline file1.txt:1]: capitalized

**Review by Estyaque — CHANGES_REQUESTED** [review]
gdsfgdf dfsgdfsgds dfgdf gfd

**Comment by Zann5547** [comment]
rtrttrytrr

**Review by Estyaque — APPROVED** [review]
